.gitignore

```diff
@@ -111,6 +111,6 @@ compiler/
 boilerplate/
 
 out/
-bsv/
+/bsv/
```

Comment:
why change this. `bsv/`  should work too

Reply:
@zhfnjust because it also blocks changes in patches/bsv, if no slash prefix is being used.